### PR TITLE
Add comment

### PR DESCRIPTION
**Pull request by qiuguohua**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
Should send to v3.7 as it only modified api doc.

**Comment by qiuguohua** [comment]
https://github.com/cocos/cocos-engine/issues/13888

File: cocos/core/platform/debug.ts [hunk 1]
```diff
@@ -53,11 +53,10 @@ function formatString (message?: any, ...optionalParams: any[]) {
 }
 
 /**
- * @en Outputs a message to the Cocos Creator Console (editor) or Web Console (runtime).
- * @zh 输出一条消息到 Cocos Creator 编辑器的 Console 或运行时 Web 端的 Console 中。
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
- * This gives you additional control over the format of the output.
+ * @en Outputs a message to the Cocos Creator Console (editor) or Web Console (runtime). This gives you additional control over the format of the output.
+ * @zh 输出一条消息到 Cocos Creator 编辑器的 Console 或运行时 Web 端的 Console 中。这为你提供了对输出格式的额外控制
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: missing `。` at the end of the sentence.

File: cocos/core/platform/debug.ts [hunk 2]
```diff
@@ -53,11 +53,10 @@ function formatString (message?: any, ...optionalParams: any[]) {
...
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en A JavaScript string containing zero or more substitution strings.
+ * @param optionalParams  @zh 用来替换在message中需要替换的JavaScript对象，@en JavaScript objects with which to replace substitution strings within msg.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: `，`-> `。`

File: cocos/core/platform/debug.ts [hunk 3]
```diff
@@ -72,8 +71,8 @@ export function log (message?: any, ...optionalParams: any[]) {
  * 输出警告消息到 Cocos Creator 编辑器的 Console 或运行时 Web 端的 Console 中。<br/>
  * - 在 Cocos Creator 中，警告信息显示是黄色的。<br/>
  * - 在 Chrome 中，警告信息有着黄色的图标以及黄色的消息文本。<br/>
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en A JavaScript string containing zero or more substitution strings.
+ * @param optionalParams  @zh 用来替换在message中需要替换的JavaScript对象，@en JavaScript objects with which to replace substitution strings within msg.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: The same as above.

File: cocos/core/platform/debug.ts (hunk 2 above)
Comment (minggo) [reply cocos/core/platform/debug.ts]: JavaScript objects to replace substitution strings in msg.

File: cocos/core/platform/debug.ts [hunk 4]
```diff
@@ -89,8 +88,8 @@ export function warn (message?: any, ...optionalParams: any[]) {
  * 输出错误消息到 Cocos Creator 编辑器的 Console 或运行时页面端的 Console 中。<br/>
  * - 在 Cocos Creator 中，错误信息显示是红色的。<br/>
  * - 在 Chrome 中，错误信息有红色的图标以及红色的消息文本。<br/>
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en A JavaScript string containing zero or more substitution strings.
+ * @param optionalParams  @zh 用来替换在message中需要替换的JavaScript对象，@en JavaScript objects with which to replace substitution strings within msg.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: The same as above.

File: cocos/core/platform/debug.ts [hunk 5]
```diff
@@ -102,9 +101,9 @@ export function error (message?: any, ...optionalParams: any[]) {
  * Assert the condition and output error messages if the condition is not true.
  * @zh
  * 对检查测试条件进行检查，如果条件不为 true 则输出错误消息
- * @param value - The condition to check on
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
+ * @param value @zh 需要检查的条件 @en The condition to check on
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en A JavaScript string containing zero or more substitution strings.
+ * @param optionalParams  @zh 用来替换在message中需要替换的JavaScript对象，@en JavaScript objects with which to replace substitution strings within msg.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: The same as above.

File: cocos/core/platform/debug.ts [hunk 6]
```diff
@@ -114,6 +113,7 @@ export function assert (value: any, message?: string, ...optionalParams: any[]):
 /**
  * @en Outputs a message at the "debug" log level.
  * @zh 输出一条“调试”日志等级的消息。
+ * @param @zh 输出的消息 @en The output message object.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: - `@param` -> `@param data`
- missing `。`

File: cocos/core/platform/sys.ts [hunk 7]
```diff
@@ -56,6 +56,7 @@ export const sys = {
      * Returns if the specified platform related feature is supported.
      * @zh
      * 返回指定的平台相关的特性是否支持。
+     * @param feature @zh 特性 @en Feature
```
Comment (minggo) [inline cocos/core/platform/sys.ts]: The comment is meaningless.

File: cocos/core/platform/debug.ts [hunk 8]
```diff
@@ -47,17 +47,22 @@ let ccAssert = (condition: any, message?: any, ...optionalParams: any[]) => {
 
 let ccDebug = ccLog;
 
+/**
+ * @en Formatting strings.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: Formatting strings -> Format a string.

File: cocos/core/platform/debug.ts [hunk 9]
```diff
@@ -89,8 +94,8 @@ export function warn (message?: any, ...optionalParams: any[]) {
  * 输出错误消息到 Cocos Creator 编辑器的 Console 或运行时页面端的 Console 中。<br/>
  * - 在 Cocos Creator 中，错误信息显示是红色的。<br/>
  * - 在 Chrome 中，错误信息有红色的图标以及红色的消息文本。<br/>
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en JavaScript string containing zero or more substitution strings.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: JavaScript string containing -> A JavaScript string contains

File: cocos/core/platform/debug.ts [hunk 10]
```diff
@@ -102,9 +107,9 @@ export function error (message?: any, ...optionalParams: any[]) {
  * Assert the condition and output error messages if the condition is not true.
  * @zh
  * 对检查测试条件进行检查，如果条件不为 true 则输出错误消息
- * @param value - The condition to check on
- * @param message - A JavaScript string containing zero or more substitution strings.
- * @param optionalParams - JavaScript objects with which to replace substitution strings within msg.
+ * @param value @zh 需要检查的条件。 @en The condition to check on.
+ * @param message @zh 包含零个或多个需要替换的JavaScript字符串。@en JavaScript string containing zero or more substitution strings.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: The same as above.

File: cocos/core/platform/debug.ts [hunk 11]
```diff
@@ -359,6 +365,8 @@ export enum DebugMode {
 /**
  * @en Gets error message with the error id and possible parameters.
  * @zh 通过 error id 和必要的参数来获取错误信息。
+ * @param errorId @zh 错误的ID。@en Error id.
+ * @param param @zh 输出日志。@en Output Log.
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: Log -> log

File: cocos/core/platform/debug.ts [hunk 12]
```diff
@@ -378,7 +386,7 @@ export function isDisplayStats (): boolean {
 /**
  * @en Sets whether display the FPS and debug informations on the bottom-left corner.
  * @zh 设置是否在左下角显示 FPS 和部分调试。
- * @deprecated Since v3.6, Please use profiler.showStats instead
+ * @deprecated @zh 从v3.6开始不再支持，请使用 profiler.showStats 替换。@en Since v3.6, Please use profiler.showStats instead
```
Comment (minggo) [inline cocos/core/platform/debug.ts]: missing `.`

File: cocos/core/platform/debug.ts (hunk 11 above)
Comment (minggo) [inline cocos/core/platform/debug.ts]: Log -> log

File: cocos/core/platform/screen.ts [hunk 13]
```diff
@@ -130,9 +130,9 @@ class Screen {
      * If failed to request fullscreen, another attempt will be made to request fullscreen the next time a user interaction occurs.
      * @zh 尝试使当前节点进入全屏模式，很多浏览器不允许程序触发这样的行为，必须在一个用户交互回调中才会生效。
      * 如果进入全屏失败，会在下一次用户发生交互时，再次尝试进入全屏。
-     * @param element The element to request full screen state
-     * @param onFullScreenChange callback function when full screen state changed
-     * @param onFullScreenError callback function when full screen error
+     * @param element @zh 请求全屏状态的html元素 @en The element to request full screen state
+     * @param onFullScreenChange @zh 全屏状态改变的回调函数 @en callback function when full screen state changed
+     * @param onFullScreenError @zh 全屏错误的回调函数 @en callback function when full screen error
```
Comment (minggo) [inline cocos/core/platform/screen.ts]: The same as above. And also miss `。`.

File: cocos/core/platform/sys.ts [hunk 14]
```diff
@@ -56,84 +56,85 @@ export const sys = {
      * Returns if the specified platform related feature is supported.
      * @zh
      * 返回指定的平台相关的特性是否支持。
+     * @param feature @zh 系统的特性。 @en Features of the system.
```
Comment (minggo) [inline cocos/core/platform/sys.ts]: Not `features`, it just check a feature.

Comment (minggo) [inline cocos/core/platform/sys.ts]: Not `features`, it just check a feature.

File: cocos/core/platform/sys.ts [hunk 15]
```diff
@@ -212,31 +213,32 @@ export const sys = {
...
+     * @en It is a local storage component based on HTML5 localStorage API, on web platform, it's equal to window.localStorage.
+     * @zh HTML5 标准中的 localStorage 的本地存储功能，在 Web 端等价于 window.localStorage。
      */
     localStorage: {} as Storage,
 
     /**
      * @en Get the network type of current device, return `sys.NetworkType.LAN` if failure.
-     * @zh 获取当前设备的网络类型, 如果网络类型无法获取，默认将返回 `sys.NetworkType.LAN`
+     * @zh 获取当前设备的网络类型, 如果网络类型无法获取，默认将返回 `sys.NetworkType.LAN`。
+     * @returns LAN or WWAN.
```
Comment (minggo) [inline cocos/core/platform/sys.ts]: Need `zh` too.

File: cocos/core/platform/visible-rect.ts [hunk 16]
```diff
@@ -28,67 +28,82 @@ import { Rect } from '../math';
...
-     * Height of the screen.
+     * @zh 屏幕高度。
+     * @en Height of the screen.
      */
     height: 0,
 
     /**
-     * initialize
+     * @zh 初始化函数。
+     * @en Initialization functions.
```
Comment (minggo) [inline cocos/core/platform/visible-rect.ts]: functions -> function

File: cocos/core/platform/sys.ts [hunk 17]
```diff
@@ -221,7 +221,7 @@ export const sys = {
     /**
      * @en Get the network type of current device, return `sys.NetworkType.LAN` if failure.
      * @zh 获取当前设备的网络类型, 如果网络类型无法获取，默认将返回 `sys.NetworkType.LAN`。
-     * @returns LAN or WWAN.
+     * @return @zh 返回 LAN 或 WWAN。 @en Retuan to LAN or WWAN.
```
Comment (minggo) [inline cocos/core/platform/sys.ts]: - just `LAN or WWAN`  is enough
- return -> returns

**Review by minggo — APPROVED** [review]

**Pull request merged**